### ISSUE TITLE
Comand Palette option to donwload sync

### ISSUE THREAD

**Issue by wsmelton** [shanalikhan/code-settings-sync#571]
**Describe the bug**

**Visual Studio Code Version :** Any
**Code Settings Sync Version :** 2.9.2
**Operating System :** Windows any
**Occurs On:** Download
**Proxy Enabled:** [ No ]
**Gist Id:** [ multiple ]

Just install VS Code and then add the Sync extension. I reload and then want to only configure it to download settings from my Gist. I use the command palette to select the download and all it does is open a browser window to GitHub profile. 

There is now no open to actually configure anything with the extension using the command palette.

**Comment by wsmelton** [comment]
The image that is shown in the readme of this repository is no longer what you receive in VS Code now.

![image](https://user-images.githubusercontent.com/11204251/41436459-572dee28-6fe7-11e8-9989-47b5e641de7c.png)

This is what I receive, and had to do a screen record and pause to just be able to read it completely....as soon as you hit enter I get a browser window and this message is not actually readable:
![image](https://user-images.githubusercontent.com/11204251/41436531-8fb6670c-6fe7-11e8-86db-87a1bc11da3e.png)

**Comment by wsmelton** [comment]
It also does not appear to honor having the gist configuration manually added to my settings.json folder. Even after reload I try to download and it still wants me to try and create a token....so how are we to get it setup now?

**Comment by fernandofleury** [comment]
We can't. it's broken atm

**Comment by fernandofleury** [comment]
On osx you can go manually to `/Users/youruser/Library/Application Support/Code/User` and create a file named `syncLocalSettings.json` with the following structure:

```
{"ignoreUploadFiles":["projects.json","projects_cache_vscode.json","projects_cache_git.json","projects_cache_svn.json","gpm_projects.json","gpm-recentItems.json"],"ignoreUploadFolders":["workspaceStorage"],"replaceCodeSettings":{},"gistDescription":"Visual Studio Code Settings Sync Gist","version":292,"token":"your gh token","downloadPublicGist":false,"supportedFileExtensions":["json","code-snippets"]}
```

**Comment by pertrai1** [comment]
@fernandofleury thanks. that hack worked for me

**Comment by wsmelton** [comment]
So windows users just have to wait and I guess use other means to sync?

**Comment by wsmelton** [comment]
So for Windows do the following:

1. Open PowerShell prompt
2. Run:
VS Code: `New-Item "$env:USERPROFILE\AppData\Roaming\Code\User\syncLocalSettings.json"`
Insiders: `New-Item "$env:USERPROFILE\AppData\Roaming\Code - Insiders\User\syncLocalSettings.json"`
3. Open file in Notepad: 
VS Code: `notepad "$env:USERPROFILE\AppData\Roaming\Code\User\syncLocalSettings.json"`
Insiders: `notepad "$env:USERPROFILE\AppData\Roaming\Code - Insiders\User\syncLocalSettings.json"`
4. Paste contents below:
```
{"ignoreUploadFiles":["projects.json","projects_cache_vscode.json","projects_cache_git.json","projects_cache_svn.json","gpm_projects.json","gpm-recentItems.json"],"ignoreUploadFolders":["workspaceStorage"],"replaceCodeSettings":{},"gistDescription":"Visual Studio Code Settings Sync Gist","version":292,"token":"your gh token","downloadPublicGist":false,"supportedFileExtensions":["json","code-snippets"]}
```


```

**Comment by wsmelton** [comment]
Actually I just tried the above process on Insiders and it does not work, but I no longer get a browser popping open either so it might be fixed on latest release of the extension.

**Comment by fernandofleury** [comment]
@wsmelton I did the manual file creation for insiders on OSX and it worked for me

**Comment by fernandofleury** [comment]
@shanalikhan you are totally right. at least we are providing some fallback while it isn't fixed
